ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiLayeredWorldState.java

```diff
@@ -37,30 +37,31 @@
 /** A World State backed first by trie log layer and then by another world state. */
 public class BonsaiLayeredWorldState implements MutableWorldState, BonsaiWorldView, WorldState {
 
-  private final BonsaiWorldStateArchive worldStateArchive;
-  private final BonsaiWorldView parent;
+  private Optional<BonsaiWorldView> parent;
```

Comment:
Is this being re-worked so as only used for blocks that were before the persisted state?  (eg persisted #100, this is a trie log for #99)  In that case `parent` may not be the best name.  perhaps `nextTrieLog`

Reply:
So we can have a layer or a persistate Worldstate so I renamed it to `nextWorldView` because those are both BonsaiWorldView.

Layer9 (parent = = Layer10), Layer10 (parent= = Layer11), Layer11 (parent==PersistedWorldState)